app/forms/idv/profile_form.rb

```diff
@@ -12,19 +12,27 @@ def self.model_name
     validate :ssn_is_unique, :dob_is_sane
 
     delegate :user_id, :first_name, :last_name, :phone, :email, :dob, :ssn, :address1,
-             :address2, :city, :state, :zipcode, to: :profile
+             :address2, :city, :state, :zipcode, to: :pii_attributes
 
     def initialize(params, user)
       @user = user
-      profile.attributes = params.select { |key, _val| respond_to?(key.to_sym) }
+      params.each do |key, value|
```

Comment:
I might consider putting this activity into a method rather than `initialize` -- seems like a lot of data parsing for calling `.new` All I usually expect to see in `initialize` is ivar assignment